src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -55,11 +55,14 @@
 
 TEST_RESOURCE_PURGE_TIMEOUT = timedelta(hours=1)
 TEST_NIGHTLY_CI_RESOURCES_PURGE_TIMEOUT = timedelta(hours=3)  # Buffer for debugging nightly integration test runs
+# See https://docs.databricks.com/en/jobs/parameter-value-references.html#supported-value-references
 EXTRA_TASK_PARAMS = {
     "job_id": "{{job_id}}",
     "run_id": "{{run_id}}",
+    "start_time": "{{job.start_time.iso_datetime}}",
     "attempt": "{{job.repair_count}}",
     "parent_run_id": "{{parent_run_id}}",
+    "workspace_id": "{{workspace.id}}",
```

Comment:
@nfx: Note this new parameter I added to avoid the cli call in `WorkflowRunRecorder` to get the workspace ID

Reply:
where the CLI call is coming from? it's RuntimeContext, CLI can't reach it by design.

Reply:
Meant API call: `self._ws.get_workspace_id()`